gemelli/preprocessing.py

```diff
@@ -8,10 +8,11 @@
 
 import warnings
 import numpy as np
+from biom import Table
 from .base import _BaseConstruct
 
 
-def rclr(T):
+def tensor_rclr(T):
```

Comment:
Same as above, be careful with name changes.

Reply:
these have been changed to `matrix_rclr` and `tensor_rclr`